.github/workflows/ci.yml

```diff
@@ -102,6 +102,12 @@ jobs:
             dependencies: highest
             codecov: true
             php-ini-values: assert.exception=1, zend.assertions=1, opcache.enable=1, opcache.enable_cli=1, opcache.optimization_level=-1, opcache.jit_buffer_size=4096M, opcache.jit=1205
+          - os: ubuntu-latest
+            php-version: "8.4"
+            tools: composer:snapshot
+            dependencies: highest
+            codecov: true
+            php-ini-values: zend.assertions=1, opcache.enable=1, opcache.enable_cli=1, opcache.optimization_level=-1, opcache.jit_buffer_size=4096M, opcache.jit=1205
```

Comment:
JIT settings got changes https://php.watch/versions/8.4/opcache-jit-ini-default-changes

But looks `pcov` is not [updated](https://github.com/krakjoe/pcov/pull/111) or `codecov` is not compatible with JIT as there's in logs

```
Run composer self-update --snapshot
PHP Warning:  JIT is incompatible with third party extensions that override zend_execute_ex(). JIT disabled. in Unknown on line 0
```

Reply:
We don't need to run code coverage on PHP 8.4 right now, if that helps. I don't think it's necessary to test with JIT either.

Reply:
Thank you, it was pcov which ruined JIT, also adjusted variables passed to job!